pkg/kapp/clusterapply/changes_view.go

```diff
@@ -93,16 +89,13 @@ func (v *ChangesView) Print(ui ui.UI) {
 		}
 
 		if resource.IsProvisioned() {
-			condVal := cmdcore.NewConditionsValue(resource.Status())
 
 			row = append(row,
 				// TODO erroneously colors empty value
```

Comment:
We could remove this and other similar TODOs since we have removed the column itself.

Reply:
Removed